### ISSUE TITLE
Queries are broken if translates is called multiple times

### ISSUE THREAD

**Issue by mrbrdo** [shioyama/mobility#564]
The query scopes (Klass.i18n) stop working correctly if translates is called twice in a model class:
```ruby
class Model < ActiveRecord::Base
  translates :name 
  translates :name 
end
```
(not that it makes sense to do that, but it can happen due to class reloading etc)
The scope then generates queries like: `WHERE "table_translations_en"."name" = 'Name' AND "table_translations_en"."name" IS NULL`, so it never returns any records.

The issue is that `translates` does an `include Mobility.translations_class.new` every time, so it will add two `#<Translations (Mobility::Backends::ActiveRecord::Table) @names=name>` into Klass.ancestors. Then the query patches enumerate the modules with `scope.model.ancestors.grep(::Mobility::Translations)` and this will return 2 instances instead of 1, which results in adding bogus conditions to the query.

**Comment by shioyama** [comment]
Ok haven't tried to reproduce but this seems like a genuine bug; `translates` should be idempotent.

**Comment by shioyama** [comment]
This is a bug, fix in #577. The fix is to the Query plugin, so it fixes the problem at source rather than in `translates` (which is basically just an alias to `include Mobility::Translations.new(...)`).

**Comment by shioyama** [comment]
This is fixed in 1.2.7.
